src/utils/sources-tree.js

```diff
@@ -73,14 +73,31 @@ function createParentMap(tree: any): WeakMap<any, any> {
  * @memberof utils/sources-tree
  * @static
  */
+function getFilenameFromPath(pathname) {
+  let filename = "";
+  if (pathname) {
```

Comment:
Hi @clarkbw 

From those tests, it seems pathname could be `undefined`, so I use `if (pathname)` to check if pathname is defined.